File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.length() != t.length()) 
            return false;
            
        int count[26] = {0};
        for (int i = 0; i < s.length(); i++) {
            count[s[i] - 'a']++; // Increment for string s
            count[t[i] - 'a']--; // Decrement for string t
        }

        for (int i = 0; i < 26; i++) {
            if (count[i] != 0) return false;
        }
    
        return true;
    }
};

File: Data Structures & Algorithms/k-closest-points-to-origin/submission-0.py
class Solution:
    def kClosest(self, points: List[List[int]], k: int) -> List[List[int]]:
        heap = []
        for point in points:
            x1, y1 = point[0], point[1]
            dist = math.sqrt(((x1) ** 2) + ((y1) ** 2))
            heapq.heappush(heap, (dist, [x1, y1]))
        res = []
        count = 0
        while count < k:
            res.append(heapq.heappop(heap)[1])
            count += 1
        return res